### PR TITLE
catppuccin-papirus-folders: init at unstable-2023-02-03

### PR DESCRIPTION
**Pull request by rubyowo**
###### Description of changes
Soothing pastel theme for Papirus Icon Theme folders. A Catppuccin port for https://github.com/PapirusDevelopmentTeam/papirus-folders.
Home Page: https://github.com/catppuccin/papirus-folders

###### Things done
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rubyowo** [comment]
@SuperSandro2000 I've messed up quite a bit, wow force-pushing is dangerous, how unsurprising.
My original PR was: #207211

**Comment by lilyinstarlight** [comment]
The rebuild counts for this PR are 0. Can you add it back to `pkgs/top-level/all-packages.nix` like before?

**Comment by rubyowo** [comment]
hmm guess i just forgot. will be quick

**Review by lilyinstarlight — APPROVED** [review]
Diff looks good to me (as with previous PR before it got hecked) and it builds successfully with working icons directories

Also just as a quick note, this would shadow the `Papirus` theme if installed, but that seems to be intended behavior from upstream so it's fine (and it's not something I think you need to change)

Thanks for working on this!

---

Result of `nixpkgs-review pr 214372` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>catppuccin-papirus-folders</li>
  </ul>
</details>

File: pkgs/data/icons/catppuccin-papirus-folders/default.nix [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchFromGitHub,
+  gtk3,
+  papirus-icon-theme,
+  flavor ? "mocha",
+  accent ? "blue",
+  ...
```
Comment (SuperSandro2000) [inline pkgs/data/icons/catppuccin-papirus-folders/default.nix:9]:
```suggestion
  accent ? "blue"
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request merged**